### PR TITLE
APP-2279 Add storybook package to PRIME.

### PR DESCRIPTION
**Pull request by ethanlook**
This PR adds a new storybook for the new `@viamrobotics/prime-core` package and eventually the `@viamrobotics/prime-blocks` package. As of now, this storybook is not published anywhere.

## Change log

- Create new private `storybook` package
- Add story for `Badge`

## Testing

- `check` passes

## Review requests

Pull down the branch and try `pnpm -C packages/storybook storybook`.

### CODE REVIEW

File: README.md
Comment (ethanlook) [inline README.md:1]: For some reason, the remark GFM library doesn't play well with this table and I couldn't figure out why. Please holler if you'll miss it!

**Review by mcous — COMMENTED** [review]
Just tested per instructions in README / PR. Storybook is throwing a couple warnings, noted below.

I got tripped up by the fact that `packages/core` needs to be built first before Storybook will work. I think that's a bit of a barrier to development, and it would be nice to make things more straightforward. 

Perhaps we should move more development scripts, like storybook, to the workspace (repo) level so that we can coordinate between packages? I think this would also be nice for `lint`, `format`, etc. so we're not running `pnpm --recursive` ourselves

**Review by mcous — APPROVED** [review]
Storybook looks good and works. A couple non-essential notes to improve the DX either in this PR or in the future

File: packages/storybook/.storybook/manager-head.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+/>
+<link
+  rel="preconnect"
+  href="https://fonts.gstatic.com"
+  crossorigin
+/>
+<link
+  href="https://fonts.googleapis.com/css2?family=Public+Sans:ital,wght@0,400;0,500;0,600;0,700;1,400&display=swap"
+  rel="stylesheet"
+/>
```
[the remark this answers is not in the document]
Comment (ethanlook) [reply packages/storybook/.storybook/manager-head.html:13]: Nice, yeah this was boilerplate from Storybook.

File: packages/storybook/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@viamrobotics/prime-storybook",
```
[the remark this answers is not in the document]
Comment (ethanlook) [reply packages/storybook/package.json:2]: Great catch!

File: packages/storybook/package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    "@sveltejs/package": "^2.0.0",
+    "@types/node": "^20.4.5",
+    "@typescript-eslint/eslint-plugin": "^5.45.0",
+    "@typescript-eslint/parser": "^5.45.0",
+    "@viamrobotics/eslint-config": "^0.1.0",
+    "@viamrobotics/prettier-config": "^0.1.0",
+    "@viamrobotics/prime-core": "workspace:../core",
+    "@viamrobotics/typescript-config": "^0.0.3",
+    "autoprefixer": "^10.4.14",
+    "eslint": "^8.39.0",
```
[the remark this answers is not in the document]
Comment (ethanlook) [reply packages/storybook/package.json:34]: Totally agreed, once the dust settles I will put up a PR to put common `devDependencies` in the repo-level `package.json`.

**Pull request merged**